debug/org.eclipse.debug.ui/ui/org/eclipse/debug/internal/ui/actions/AbstractEnableAllActionDelegate.java

```diff
@@ -0,0 +1,86 @@
+package org.eclipse.debug.internal.ui.actions;
+
+import org.eclipse.jface.action.IAction;
+import org.eclipse.jface.viewers.ISelection;
+import org.eclipse.swt.widgets.Event;
+import org.eclipse.ui.IActionDelegate2;
+import org.eclipse.ui.IViewActionDelegate;
+import org.eclipse.ui.IViewPart;
+import org.eclipse.ui.IWorkbenchWindow;
+import org.eclipse.ui.IWorkbenchWindowActionDelegate;
+
+public abstract class AbstractEnableAllActionDelegate
+		implements IViewActionDelegate, IActionDelegate2, IWorkbenchWindowActionDelegate {
+
+	private IAction fAction;
+
+	/**
+	 * Needed for reflective creation
```

Comment:
a public contstructor is autocreated if no other constructor is there

Reply:
okay